assignment/README.md

```diff
@@ -1,65 +1,70 @@
+
+| View                 | Map                                                                     | Sidebar                                                  |
+| -------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------- |
+| Syracuse at-a-glance | Zoomed-out display of 22 inspection districts spanning all of the city. | Summary of each district's risk.                        |
+| Individual district  | Zoom in to one the selected district. Parcels colored by risk level.    | A sorted list of the parcels in the selected district. |
+| Parcel detail        | Further zoom in to the individual parcel.                               | A dashboard display of parcel details.               |
 
 ## Anticipated difficulties
 
-Thinking about weaknesses can be useful. What do you anticipate being
-most difficult about this project? How will you attempt to cope with
-these difficulties? For example, asynchronous behavior (ajax, events)
-are hard to use and think about. Global variables are a strategy for
-coping with that difficulty by breaking data out of the asynchronous
-context.
+#### Thinking about weaknesses can be useful. What do you anticipate being most difficult about this project? How will you attempt to cope with these difficulties? For example, asynchronous behavior (ajax, events) are hard to use and think about. Global variables are a strategy for coping with that difficulty by breaking data out of the asynchronous context.
+
+- Interactive display and styling of map layers will be a hurdle. Depending on the action of the user, certain layers will have to be hidden, displayed, or styled in a certain way. The sidebar will also have to reflect the current state of the map.  
+- The city of Syracuse has 40,000+ parcels. Because it is difficult to display this volume of polygons, only the relevant portion of this dataset should be served when the user asks for it. Server-side scripting or asynchronous JS may be necessary.
 
 ## Missing pieces
 
-We've only managed to scratch the surface of the available technologies
-by which you could construct an application. What use-cases haven't we covered
-that you think would be useful? What technologies not covered seem exciting to
-you (you don't necessarily have to fully understand what they're for,
-this is a chance for you to get our help interpreting a technology's
-purpose/usage).
+#### We've only managed to scratch the surface of the available technologies by which you could construct an application. What use-cases haven't we covered that you think would be useful? What technologies not covered seem exciting to you (you don't necessarily have to fully understand what they're for, this is a chance for you to get our help interpreting a technology's purpose/usage).
+
+- How can Javascript be used to query a database and rapidly return only relevant information? 
```

Comment:
If there are only a couple of tables and you know exactly how queries will be structured, a tiny `flask` (python) script + `sqlalchemy` (or other relational database library) should be up to the task. We can get started on something like this together if you like